### DISCUSSION TITLE
Hmm... this logo looks a little familiar...

### DISCUSSION THREAD

**Discussion by TheDuggem** [supermerill/SuperSlicer#3909] [Community contribution]
I was working on my wife's computer and saw this icon in one of her browser windows:

![image](https://github.com/supermerill/SuperSlicer/assets/5025662/a7e69a85-f0e8-49fc-9ab3-6a3788eb63df)

It's for a web-based PDF program named OrbitNote that was released in January of 2022.  I wouldn't say it infringes, but the similarity caught my attention.

**Comment by SirOink** [comment]
sadly supermerill ain't some bald billionaire that could protect his IP anyway 
does make you wonder if their young graphics designer owns a voron though eh

**Reply by NameOfTheDragon** [reply]
It is probably further from the SS logo than the SS logo is from Prusa Slicer logo, so "infringement" would be an interesting discussion.

**Comment by EvilGremlin** [comment]
In the first place, logo idea of Slic3r is very generic.
I bet you can find same logo popping up in all colors all around the world for over a century.
